Examples/Scripts/SetUpHCPPipeline.sh

```diff
@@ -22,7 +22,7 @@ fi
 ## Edit this section: set up other environment variables
 export MSMBINDIR="${HOME}/pipeline_tools/MSM"
 export MATLAB_COMPILER_RUNTIME=/export/matlab/MCR/R2017b/v93
-export FSL_FIXDIR=/usr/local/fix
+# export FSL_FIXDIR=/usr/local/fix # only needed for legacy fix
```

Comment:
Why is this commented out?   Does this PR eliminate support altogether for "legacy fix"?

Reply:
The current state defaults to legacy fix only if `FSL_FIXDIR` is set.  So, for users that previously had it set up, it will use legacy fix, while users with a new copy of `SetUp` will use pyfix unless they uncomment this.

Reply:
So, could the comment instead say: `# Uncomment to use legacy (Matlab-based) fix; otherwise pyfix will be used`

Reply:
Control of which type of FIX to use has moved to the FIX code itself.  If nothing is set then it is pyfix (previously this was an error).